src/pages/Landing/__snapshots__/index.test.tsx.snap

```diff
@@ -4512,7 +4512,7 @@ exports[`disable nft on landing page renders nft information and card 1`] = `
       </div>
       <a
         class="9 c58"
-        href="https://wallet.uniswap.org/"
+        href="https://wallet.uniswap.org/?utm_source=home_page&utm_medium=webapp&utm_campaign=wallet_microsite&utm_id=1"
```

Comment:
Can you make sure both the iOS and microsite are tested in a snapshot, by mocking the isIOS hook?